core/domain/suggestion_services_test.py

```diff
@@ -88,6 +88,33 @@ def setUp(self):
         self.save_new_valid_exploration(
             self.target_id, self.author_id, category='Algebra')
 
+    def assert_suggestion_status(self, suggestion_id, status):
+        """Assert the status of the suggestion with suggestion_id."""
+        suggestion = suggestion_services.get_suggestion_by_id(suggestion_id)
+        self.assertEqual(suggestion.status, status)
+
+    def accept_suggestion_setup_with_mocks(
```

Comment:
def mock_accept_suggestion

(Callers do not care what you do inside this function with regards to setup etc., so you're exposing too much internal detail in the method name.)